Makefile

```diff
@@ -54,6 +54,9 @@ init: node_modules ;
 # Use building English language as a check to see if everything works
 test: en ;
 
+lint:
```

Comment:
> introduce an all target similar to the current default

For some reason I thought that's the way it was, but I see that's not quite the case. Yes that is what I'd recommend too: `default` being the bare bones minimum to go from zero to working build, and `all` being a sort of kitchen sink build. Meanwhile `test` should probably call `lint` as a prerequisite:

    test: lint en ;